youtube2zim/scraper.py

```diff
@@ -792,6 +792,16 @@ def make_html_files(self, actual_videos_ids):
             - videos/<videoId>/video.jpg            template
         """
 
+        def remove_unused_videos(used_videos):
```

Comment:
could you use `videos` variable name? It would be in its own block and feels more natural as that's what we used.

Reply:
Done